web/src/beta/hooks/useAssetUploader/hooks.tsx

```diff
@@ -9,20 +9,23 @@ export default ({
   workspaceId,
   onAssetSelect,
   assetType,
+  fileFormat,
 }: {
   workspaceId?: string;
   onAssetSelect?: (inputValue?: string, name?: string) => void;
   assetType?: string;
+  fileFormat?: "CSV" | "GeoJSON" | "KML" | "CZML";
 }) => {
   const { useCreateAssets } = useAssetsFetcher();
 
+  const fileFormatAccepted = fileFormat ? `.${fileFormat?.toLocaleLowerCase()}` : FILE_FORMATS;
```

Comment:
```
  const acceptedFileFormats = useMemo(()=> fileFormat ? `.${fileFormat.toLocaleLowerCase()}` : FILE_FORMATS, []);
```

Reply:
Then again, if this is only being used in the acceptedExtension, moving it in to that useMemo is best 🙏🏻 up to you